experimental/packages/otlp-transformer/test/logs.test.ts

```diff
@@ -78,78 +95,125 @@ function createExpectedLogJson(useHex: boolean): IExportLogsServiceRequest {
-        resource: resource_1,
-        instrumentationScope: scope_2,
-      };
-      log_2_1_1 = {
-        ...log_fragment_1,
-        resource: resource_2,
-        instrumentationScope: scope_1,
-      };
+        scopeLogs: [
+          {
+            scope: { name: 'scope_name_1', version: '0.1.0' },
+            logRecords: [
+              {
+                timeUnixNano: 1680253513123241700,
+                observedTimeUnixNano: 1683526948965142800,
+                severityNumber: ESeverityNumber.SEVERITY_NUMBER_ERROR,
+                severityText: 'error',
+                body: { stringValue: 'some_log_body' },
+
+                attributes: [
+                  {
+                    key: 'some-attribute',
+                    value: { stringValue: 'some attribute value' },
+                  },
+                ],
+                droppedAttributesCount: 0,
+                flags: 1,
+                traceId: traceId,
+                spanId: spanId,
+              },
+            ],
+            schemaUrl: 'http://url.to.schema',
+          },
+        ],
+      },
+    ],
+  };
+}
+
+describe('Logs', () => {
+  let resource_1: Resource;
+  let resource_2: Resource;
+  let scope_1: InstrumentationScope;
+  let scope_2: InstrumentationScope;
+  let log_1_1_1: ReadableLogRecord;
+  let log_1_1_2: ReadableLogRecord;
+  let log_1_2_1: ReadableLogRecord;
+  let log_2_1_1: ReadableLogRecord;
+
+  beforeEach(() => {
+    resource_1 = new Resource({
+      'resource-attribute': 'some attribute value',
+    });
+    resource_2 = new Resource({
+      'resource-attribute': 'another attribute value',
     });
+    scope_1 = {
+      name: 'scope_name_1',
+      version: '0.1.0',
+      schemaUrl: 'http://url.to.schema',
+    };
+    scope_2 = {
+      name: 'scope_name_2',
+    };
+    const log_fragment_1 = {
+      hrTime: [1680253513, 123241635] as HrTime,
+      hrTimeObserved: [1683526948, 965142784] as HrTime,
+      attributes: {
+        'some-attribute': 'some attribute value',
+      },
+      droppedAttributesCount: 0,
+      severityNumber: SeverityNumber.ERROR,
+      severityText: 'error',
+      body: 'some_log_body',
+      spanContext: {
+        spanId: '0000000000000002',
+        traceFlags: TraceFlags.SAMPLED,
+        traceId: '00000000000000000000000000000001',
+      },
+    };
+    const log_fragment_2 = {
+      hrTime: [1680253797, 687038506] as HrTime,
+      hrTimeObserved: [1680253797, 687038506] as HrTime,
+      attributes: {
+        'another-attribute': 'another attribute value',
+      },
+      droppedAttributesCount: 0,
+    };
+    log_1_1_1 = {
+      ...log_fragment_1,
+      resource: resource_1,
+      instrumentationScope: scope_1,
+    };
+    log_1_1_2 = {
+      ...log_fragment_2,
+      resource: resource_1,
+      instrumentationScope: scope_1,
+    };
+    log_1_2_1 = {
+      ...log_fragment_1,
+      resource: resource_1,
+      instrumentationScope: scope_2,
+    };
+    log_2_1_1 = {
```

Comment:
Yep that's true. Working with them also was not very ergonomic. These lines were just moved up one `describe()` block since I added more tests here; they only appear to be new in the diff due to the new indentation.

Added a comment how these are structured. :slightly_smiling_face: 
[4f80ada](https://github.com/open-telemetry/opentelemetry-js/pull/4542/commits/4f80adaa29d6d00a2d9238c7d67ebd46f2877a96)